### ISSUE TITLE
Prototyp in Wishlist Repo einbauen

### ISSUE THREAD

**Issue by Erdnaf** [FUB-HCC/20-SWP-CodingOpenness#205]
# Bounty
Formulierung für das Repo finden
## Scope

-

## Deliverables

-

## Gain for the project

-

## Roles

bounty worker: @Erdanf 
bounty reviewer: @username

## Issue creation checklist

- [x] I have attached one of the team labels `Fragebogen`, `Technik`, `Projektwebseite`, `Vergleich`, `Organisation` to this bounty
- [x] I have attached one of the priority labels `Blocker`, `High`, `Medium`, `Low` to this bounty
- [x] I have attached one of the `size-` labels to this bounty
- [x] I have added this bounty to bounty board here https://github.com/FUB-HCC/20-SWP-CodingOpenness/projects/1

**Comment by Erdnaf** [comment]
We have worked on this Topic and develloped a design Prototype that can be found here:
https://www.figma.com/proto/sopIMcNqDFLYZ6sf3rVtbt/Colapsibile-%2B-Icons?node-id=0%3A1&scaling=scale-down

First there are some Icons to the initial view, seen on the first page of the prototype:

- Icon for legal only text
- Icons for the three main Functiones mentioned in the Data Privacy Document
- Icons for the Android Permissions

On the following Pages we've added a small presentation on unfoldable areas. The entries 1 and 5(a) are clickable.

I hope with this Prasentation there is more clearity about what this Issues aim is.
I would love to here more feedback.

Das ist mein Vorschlag, bitte um Review. Ich hab extra ein bischen offen gelassen, wer das 'we' im Text ist, ich hoffe das ist okay so.

**Comment by linuxhelf** [comment]
@Erdanf 
sieht schonmal gut aus
ein paar Kleinigkeiten:

- functions statt Function**e**s
- pages klein schreiben
- pr**e**sentation statt Prasentation
- clarity statt clearity
- hear statt here
- und ein paar weitere Wörter die groß geschrieben sind und evtl. klein sein sollten
- das "more feedback" klingt komisch, da es noch keins gibt 😄 evtl. "some feedback"?

**Comment by Erdnaf** [comment]
We have worked on this topic and develloped a design prototype that can be found here:
https://www.figma.com/proto/sopIMcNqDFLYZ6sf3rVtbt/Colapsibile-%2B-Icons?node-id=0%3A1&scaling=scale-down

First there are some icons to the initial view, seen on the first page of the prototype:

-Icon for legal only text
-Icons for the three main functions mentioned in the "Data Privacy Document"
-Icons for the Android permissions

On the following pages we've added a small presentation on unfoldable areas. The entries 1 and 5(a) are clickable.

I hope with this presentation there is more clarity about what this Issues aim is.
I would love to hear some feedback.

Danke für das Feedback, werde es gleich ins Wishlist repo schreiben
